src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -731,6 +731,34 @@ public ContainerCreation commitContainer(final String containerId,
   }
 
   @Override
+  public void renameContainer(final String containerId, final String name)
+          throws DockerException, InterruptedException {
+    WebTarget resource = resource()
+            .path("containers").path(containerId).path("rename");
+
+    if (name != null) {
```

Comment:
can the `name` actually be null? I would have thought that the [rename operation](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/) returned an error if the `name` parameter was not supplied